cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -294,30 +294,25 @@ void SparsePoseGraph::WaitForAllComputations() {
   common::MutexLocker locker(&mutex_);
   const int num_finished_scans_at_start =
       constraint_builder_.GetNumFinishedScans();
-  while (!locker.AwaitWithTimeout(
-      [this]() REQUIRES(mutex_) {
-        return static_cast<size_t>(constraint_builder_.GetNumFinishedScans()) ==
-               trajectory_nodes_.size();
-      },
-      common::FromSeconds(1.))) {
+  while (!locker.AwaitWithTimeout([this]() REQUIRES(mutex_) {
```

Comment:
This code style change seems like irrelevant to this commit message. Is it accidental?